packages/@azure/servicebus/data-plane/examples/javascript/gettingStarted/sendWithInteractiveAuth.js

```diff
@@ -0,0 +1,56 @@
+/*
+  This sample demonstrates how to create a namespace using TokenCredentials.
+
+  Setup: AAD RBAC is enabled only on the new namespaces in these regions
```

Comment:
I dont believe RBAC is a well known acronym. Let's expand this.

Also, reword this to

"Please ensure that your Azure Service Bus resource is in one of the below regions. AAD Role Based Access Control is not supported in other regions yet."

Reply:
Same for the other file